src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1009,11 +1038,6 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
-	// When changing kernel, update the active session
-	private updateActiveClientSession(clientSession: IClientSession) {
```

Comment:
This method was only used in one place and was just a simple assignment, so I removed it.